[0m06:01:16.284694 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffccd3340>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd411bd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd413ee0>]}


============================== 06:01:16.293175 | 15695438-6243-4b28-98bc-14362f823937 ==============================
[0m06:01:16.293175 [info ] [MainThread]: Running with dbt=1.7.11
[0m06:01:16.294209 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/dbt/datawarehouse', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m06:01:16.295240 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m06:01:16.298013 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.062055, "process_user_time": 1.643682, "process_kernel_time": 0.05099, "process_mem_max_rss": "97044", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m06:01:16.299013 [debug] [MainThread]: Command `dbt run` failed at 06:01:16.298799 after 0.06 seconds
[0m06:01:16.299650 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffccd3340>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd410d60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7ffffd413ee0>]}
[0m06:01:16.300118 [debug] [MainThread]: Flushing usage events
